cdap-validation/bin/ambari.py

```diff
@@ -0,0 +1,87 @@
+import urllib2
+import json
+import re
+import glob, os
+import test_helpers as helpers
+
+# Start Ambari configuration retrieval process
+def ambari_commands(host, subdir, base, cluster_info):
+    ## get ambari configs
+    manager = 'ambari' 
+    cu = get_config_urls(host, cluster_info)
+    if cluster_info['verbose'] == 2: print 'cu=%s' % (cu)
```

Comment:
You should create your own output function with 2 arguments (second optional), message and level (default to normal). This way you don't have to repeat this conditional logic, everywhere.